modules/user_accounts/ajax/getData.php

```diff
@@ -0,0 +1,141 @@
+<?php
```

Comment:
Would this be cleaner if it was implemented as a toJSON function on the page itself instead of a separate script and accessed the page with `?format=json`?

If not, can you give this script a more descriptive filename than "getData"? It's obvious that it's getting data, it's accessed via an HTTP GET request. What data is it getting?